templates/about.tpl

```diff
@@ -1,23 +1,25 @@
-% include('templates/header', title='About')
+% rebase('base', title='About')
 
 <h1>About</h1>
 <hr />
 
+% realtime_count = len([s for s in systems if s.realtime_enabled])
+% non_realtime_count = len([s for s in systems if not s.realtime_enabled])
 <p>
-  BCTracker is a browser for the <a href="https://developers.google.com/transit/gtfs">GTFS</a> static and realtime data provided by the Greater Victoria Transit System operated by <a href="https://bctransit.com/victoria">BC Transit</a>.
+  BCTracker is a browser for the <a href="https://developers.google.com/transit/gtfs">GTFS</a> static and realtime data provided by <a href="https://bctransit.com">BC Transit</a>.
   The data is presented here for the use of whoever is interested, with the goal of making it easier to browse schedules and track down buses.
+  
+  Realtime data is provided for the {{ realtime_count }} systems that currently support it, along with {{ non_realtime_count }} other systems where only schedule data is available. 
 <p>
 
 <p>
   The website (both the pages and the data processing code) is written in python using a simple web framework called Bottle.
-  The code can be found on <a href="https://github.com/bumblesquashs/bctracker-victoria">GitHub</a> if you are interested in seeing how it works or trying to run it yourself.
+  The code can be found on <a href="https://github.com/bumblesquashs/bctracker">GitHub</a> if you are interested in seeing how it works or trying to run it yourself.
 </p>
 
 <p>
   If you are curious about BCTracker, have questions, or something seems broken, contact us by emailing <a href="mailto:james@bctracker.ca">james@bctracker.ca</a>
 </p>
 
 <br />
-<i> - James & Perrin, 2020 </i>
-
-% include('templates/footer')
+<i> - James & Perrin, 2021 </i>
```

Comment:
Typo: "James": Not sure why this says James here, this is clearly from the point of view of the architect of this site... not sure this is correct

Reply:
Sorry will update it to say Bumblesquash, for clearly that is the name of the other developer